.github/workflows/.reusable_e2e_tests_oss.yml

```diff
@@ -90,6 +92,21 @@ jobs:
           docker compose -f .ci/docker-compose.yml logs
           exit $_compose_exit
 
+      - name: Start Kong hybrid
+        if: ${{ matrix.suite == 'data-plane-nodes' }}
+        timeout-minutes: 10
+        working-directory: ${{ github.workspace }}
+        env:
+          GATEWAY_IMAGE: ${{ inputs.gateway-image }}
+        run: |
+          mkdir -p hybrid
```

Comment:
Maybe we can add `chmod a+w ./hybrid` after the `mkdir`

Reply:
It worked!